app/services/ai_chat_engine.py

```
@@ -1270,6 +1270,66 @@ async def _check_rebalancing_rate_limits(self, user_id: str) -> Dict[str, Any]:
             self.logger.error("Rate limiting check failed", error=str(e), user_id=user_id)
             return {"allowed": True}  # Fail open for availability
     
+    def _format_strategy_comparison_response(self, strategy_comparison: Dict, user_id: str) -> Dict[str, Any]:
```

Ruff (0.12.2)

Unused method argument: `user_id`

(ARG002)